src/data_management/rasters/NewRaster2.js

```diff
@@ -26,6 +27,19 @@ class NewRaster2 extends Component {
   render() {
     return (
       <ManagementForm onSubmit={onSubmitExample} wizardStyle>
+        <Slushbucket
+          name="sharedWithOrganisation"
+          title="Shared With Organisation"
+          choices={[
+            { value: "abc", display: "aBc" },
+            { value: "nens", display: "nEns" },
+            { value: "home", display: "hOmbre" },
+          ]}
+          readonly={false}
+          isFetching={false}
+          placeholder={"search organisations"}
+          initial={[]}
```

Comment:
Werkt het ook zonder dit? (initial=null)

Reply:
Zonder dit gaf hij een error.
Ik denk dat het ook wel mooi is om hier een array mee te geven, want zo wordt duidelijk dat het om een array type gaat

Reply:
Maar dit maakt het het enige element dat verplicht een initial mee moet krijgen. Ik wil eigenlijk dat dat nooit hoeft bij een leeg (nieuw) formulier.

Kwestie van `const selected = value || []` bovenaan render, denk ik.

Reply:
Je hebt gelijk. Dit werkt. Done